dev-libs/libezV24/libezV24-0.1.1-r1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="library that provides an easy API to Linux serial ports"
+HOMEPAGE="http://ezv24.sourceforge.net"
+SRC_URI="mirror://sourceforge/ezv24/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~sparc ~x86"
+
+HTML_DOCS=( api-html/. )
+
+PATCHES=(
+	"${FILESDIR}"/${P}-build.patch
+	"${FILESDIR}"/${P}-test.patch
+)
+
+src_prepare() {
+	tc-export AR CC RANLIB
+	default
+	sed -i -e 's:__LINUX__:__linux__:' *.c *.h || die
+}
+
+src_install() {
+	export NO_LDCONFIG="stupid"
+	emake DESTDIR="${D}" LIBDIR="/usr/$(get_libdir)" install
```

Comment:
Some EPREFIX support maybe, while you're bumping the EAPI?

Reply:
done :)

Reply:
I'm going to guess there's a `PREFIX` variable that you need to adjust as well.

Reply:
@mgorny 
I'm sorry, can you explain. I don't know exactly what you mean.

Reply:
I mean that there's probably a variable defaulting to `/usr` where you need to append `${EPREFIX}` so it'll apply to everything. Hm, unless the library is the only thing it ever installs.

Reply:
@mgorny 
Thx for explaining and sorry for the late reply. I've looked at the Makefile and as you said there is a ```PREFIX``` variable, which i'm now also adjusting with ```EPREFIX``` too.